src/view/ChewingEditor.cpp

```diff
@@ -104,6 +104,8 @@ void ChewingEditor::execFileDialog(DialogType type)
 
 void ChewingEditor::selectImportFile()
 {
+    import_count = 0;
```

Comment:
Don't violate the pattern [Model–view–controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). In particular, `import_count` should be encapsulated within the "view" rather than the "model".